charts/render.py

```diff
@@ -328,6 +328,13 @@ def __call__(self, parser, args, values, option_string=None):
     parser.add_argument("path", help="Path to the chart directory")
     args = parser.parse_args()
 
+    api_versions = [
+        # Used by APIService (available since Kubernetes 1.10)
+        "api.registration.k8s.io",
```

Comment:
Here it's written `api.registration`, whilst in the previous approach it was `apiregistration`. Which one is correct?

Reply:
Oh right it's a typo, good catch